src/core/wb.js

```diff
@@ -571,14 +571,29 @@ Modernizr.load( [
 					// Start initialization
 					wb.init( document, componentName, selector );
 
+					// Disable MathJax's context menu to more closely mimic native MathML implementations
+					window.MathJax = {
+						options: {
+							enableMenu: false
+						}
+					};
+
 					// Load the MathML dependency. Since the polyfill is only loaded
 					// when !Modernizr.mathml, we can skip the test here.
 					Modernizr.load( [ {
-						load: "timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/MathJax.js?config=Accessible",
+
+						// Load ES6 polyfill (for IE11) and MathJax from CDNs
+						load: [
+							"timeout=500!https://polyfill.io/v3/polyfill.min.js?features=es6",
+							"timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/es5/mml-chtml.js"
+						],
 						complete: function() {
+
+							// Fall back on a local copy MathJax if CDN loading fails
+							// Note: Won't work with IE11 in isolated networks (ES6 polyfill has no local fallback)
 							Modernizr.load( [ {
-								test: window.MathJax === undefined,
-								yep: "mthjx!MathJax.js?config=Accessible"
+								test: window.MathJax.startup === undefined,
+								yep: "mthjx!mml-chtml.js"
```

Comment:
Can we try to load the old MathJax if it don't work with 3

Reply:
Just added this in... but should note that it isn't really ideal and will take a while to load in. I've implemented it as a last ditch effort to load in *some* kind of MathJax flavour.

It won't attempt to load in at the same time as MathJax 3 (since there's a risk of conflicts if both exist). It'll only try to load in if both MathJax 3 attempts (for the CDN and local versions) fail.

One way of speeding this up a bit would be to stop trying to load MathJax 3 from a CDN and just go straight to checking the local MathJax 3 dependency from the get-go. That'd skip one "step" in the event that MathJax 2 needs to be fallen back on.